front/public/cards/ld.json

```diff
@@ -0,0 +1,36 @@
+{
+    "ppcalls": [
+        {
+            "message": "<p>Data shown from the <a href='https://viz.stjude.cloud/cancer-survivorship/visualization/st-jude-survivorship-portal~18'>St. Jude Survivorship Portal<a/>.</p>",
+            "urlparam": "?block=on&genome=hg38&ldfile=local,files/hg38/sjlife/ld/CEU.gz",
+            "runargs": {
+                "parseurl": true,
+                "block": true,
+                "nobox": 1,
+                "noheader": 1,
+                "genome": "hg38",
+                "nativetracks": "Refgene",
+                "tracks": [
+                    {
+                        "type": "ld",
+                        "name": "European ancestry",
+                        "file": "files/hg38/sjlife/ld/CEU.gz",
```

Comment:
please replace this actual dataset file with a slice file and load it from tp/proteinpaint_demo/hg38/LD/